modules/cluster-logging-deploying-about.adoc

```diff
@@ -161,14 +161,16 @@ spec:
   logStore:
     type: "elasticsearch"
     elasticsearch:
-      nodeCount: 2
+      nodeCount: 3
```

Comment:
Could we add  retentionPolicy here?
      retentionPolicy: <1>
        application:
          maxAge: 1d
        infra:
          maxAge: 7d
        audit:
          maxAge: 7d

Reply:
Will be added by https://github.com/openshift/openshift-docs/pull/22403/files#diff-2230f32010df8cbe4fb588517ca026f7